awswrangler/s3/_read.py

```diff
@@ -71,11 +71,11 @@ def _extract_partitions_from_path(path_root: str, path: str) -> Dict[str, str]:
     if path_root not in path:
         raise exceptions.InvalidArgumentValue(f"Object {path} is not under the root path ({path_root}).")
     path_wo_filename: str = path.rpartition("/")[0] + "/"
-    path_wo_prefix: str = path_wo_filename.replace(f"{path_root}/", "")
-    dirs: Tuple[str, ...] = tuple(x for x in path_wo_prefix.split("/") if (x != "") and (x.count("=") == 1))
+    path_wo_prefix: str = path_wo_filename.replace(f"{path_root}", "")
+    dirs: Tuple[str, ...] = tuple(x for x in path_wo_prefix.split("/") if x != "")
```

Comment:
Don't we still want to check if `"=" in x`?

Reply:
Oof, good catch